homeworks/vladyslav.makedon_Vladmaked/8-components-templates/Gallery-app/src/app/components/gallery/gallery.component.html

```diff
@@ -0,0 +1,21 @@
+<header>
+    <h1 class="header">Gallery</h1>
+</header>
+<ul class="gallery">
+    <li class="gallery__item" *ngFor="let image of images; let index = index">
+        <div class="gallery__image-wrapper" [class.selected]="getIsImageSelected(image)">
+            <app-gallery-item [image]="image"
+                              [selectedIndex]="index"
+                              (isLoadedImageEvent)="setImageLoadStatus($event)"
+                              (selectedImageEvent)="setSelectedImage($event)"
+                              (selectedImageIndexEvent)="setImageSelectedIndex($event)">
+            </app-gallery-item>
+            <app-loader *ngIf="!isLoadedImage" [ngStyle]="{display: isLoadedImage? 'none': 'block'}"></app-loader>
```

Comment:
You don't need [ngStyle] because if image is not loaded, node won't be created and styles won't be applied.

So just remove the extra attribute 😄